client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,34 @@
-import React from 'react';
+import React, {useState} from 'react'
+import {Link} from 'react-router-dom'
 
+
+const Movie = (props) => {
+  const [movie, setMovie] = useState()
+  const id = props.match.params.id
+}
 const MovieCard = props => {
-  return;
+  const {title, director, metascore, stars} = movie
```

Comment:
We would need to define the movie variable, above we could write `const { movie } = props`.

Reply:
const movie = props.move;